persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageDAOImpl.kt

```diff
@@ -47,92 +47,91 @@ class MessageDAOImpl(private val queries: MessagesQueries) : MessageDAO {
 
     @Suppress("ComplexMethod", "LongMethod")
     private fun insertInDB(message: MessageEntity) {
-        queries.transaction {
```

Comment:
If we call `insertInDB` directly (which is what `insertMessage(MessageEntity)` does), there are no transactions at all, no?

Maybe we need to add the transaction to `insertMessage(MessageEntity)`

Reply:
Good catch, I added a comment/doc as well for function callers